src/classes/STG_Panel.cls

```diff
@@ -467,4 +448,29 @@ public with sharing virtual class STG_Panel {
         return USER_UserService.isSysAdmin(new List<User>{(new User(Id = runningUserId, ProfileId = runningUserProfileId))}).get(runningUserId);
 
     }
+
+    /*******************************************************************************************************
+    * @description Get the label of the picklist value
+    * @param objectName The Sobject name of the picklist field
+    * @param fieldName The field name of the picklist field
+    * @param apiValue The value of the picklist field
+    * @return String The label value of the picklist field
+    */
+    public static String getPicklistLabelFromValue(String objectName, String fieldName, String apiValue) {
```

Comment:
There is logic in the NPSP Settings > Donations > Contact Roles that verifies assignment. For example, a HH Member Contact Role cannot be the same as the Matched Donor Role or Honoree Role or Notification Recipient Role. These rules would need to be retested for both translated org user and the English lang user.
@ravensisco , @andrewyu-salesforce